src/Nethermind/Nethermind.Db/MetadataDbKeys.cs

```diff
@@ -13,5 +13,7 @@ public static class MetadataDbKeys
         public const int BeaconSyncPivotNumber = 6;
         public const int LowestInsertedBeaconHeaderHash = 7;
         public const int FirstPoSHash = 8;
+        public const int UpdatedPivotHash = 9;
+        public const int UpdatedPivotNumber = 10;
```

Comment:
I dislike that there are 2 keys, we should pack this into 1 value, having first 8bytes being number and later 32 bytes being a hash.

Or just using RLP.

Reply:
There will be really small advantage in number of writes/reads - we are writing it only once per node life and reading it only when there was a restart during sync. So performance is definitely not an issue here. And after this change there will be a lack of consistency - we will have:
```
        public const int TerminalPoWHash = 1;
        public const int TerminalPoWNumber = 2;
(...)
        public const int BeaconSyncPivotHash = 5;
        public const int BeaconSyncPivotNumber = 6;
(...)
        public const int UpdatedPivotData = 9;
```
so hashes, numbers and one combined value of number+hash.
In other values we are using separated numbers and hashes. I can change it if you want, but IMO it is better in current form.